test/integration/cart_integration_test.rb

```diff
@@ -14,29 +16,81 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
   test 'an unauthorized user can add an item to the cart' do
     item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
     visit "/items/#{item.id}"
+
     click_link_or_button('Add to Cart')
+
     within('#flash_notice') do
       assert page.has_content?('Added to Cart')
     end
+  end
+
+  test 'an unauthorized user with items in their cart can view their cart' do
+    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
+    visit "/items/#{item.id}"
+
+    click_link_or_button('Add to Cart')
     click_link_or_button('Cart')
+
     within('#cart_items') do
       assert page.has_content?('coffee')
       assert page.has_content?('1')
     end
   end
 
-  test 'an unauthorized user can add nine of the same item' do
+  test 'an unauthorized user can add up to 10 of the same item at one time' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.